lib/facebook.ex

```diff
@@ -114,6 +114,43 @@ defmodule Facebook do
 		Facebook.Graph.get(~s(/#{user_id}/permissions), fields, options)
 	end
 
+	@doc """
+	Get the number of likes for the provided page_id
+	"""
+	@spec pageLikes(page_id :: integer | String.t, access_token) :: integer
+	def pageLikes(page_id, access_token) do
+		{:json, %{"likes" => likes}} = page(page_id, access_token, ["likes"], [])
+		likes
+	end
+
+	@doc """
+	Basic page information for the provided page_id
+
+	See: https://developers.facebook.com/docs/graph-api/reference/page
+	"""
+	@spec page(page_id :: integer | String.t) :: response
+	def page(page_id) do
+		page(page_id, nil, [], [])
```

Comment:
probably here you should also accept an access_token, but if the access_token is not required (see other comment), we can leave it